src/lib/Config.ts

```diff
@@ -18,11 +18,10 @@ import {promisify} from 'util';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as os from 'os';
-import userPrompt = require('prompt');
+import * as inquirer from 'inquirer';
```

Comment:
Not essential, but I'd still prefer we be precise with our imports.

Reply:
I'm a bit divided in the case where a library has many related methods. 

When we import with `import * as inquirer from 'inquirer'`, then we get to use `inquirer.prompt(...)` in the code, which makes it clear where prompt is coming from.

When we import with `import {prompt} from 'inquirer'`, the call in the code becomes just `prompt`, which is less obvious where it comes from, unless you scan the imports. 

WDYT?